src/containers/SinglePositionWrapper/SinglePositionWrapper.tsx

```diff
@@ -296,9 +296,23 @@ export const SinglePositionWrapper: React.FC<IProps> = ({ id }) => {
   }, [walletStatus])
 
   useEffect(() => {
-    dispatch(actions.getSinglePosition(id))
+    onRefresh()
   }, [])
 
+  const onRefresh = () => {
```

Comment:
here similar question like on the create position. add refresh of balances and token price as well?